website/conferences/views.py

```diff
@@ -138,6 +155,27 @@ def add_poster_by_email(conference, message):
     if node_created and user_created:
         signals.osf4m_user_created.send(user, conference=conference, node=node)
 
+    # Send log to Keen
+    keen.add_event('osf4m', {
+        "user": {
+            "username": user.username,
+            "userEmail": message.sender_email,
```

Comment:
Please make this a subproperty of the "osf4m" key and rename it to "receivedFrom".